Base/etc/SystemServer.ini

```diff
@@ -169,3 +169,11 @@ AcceptSocketConnections=1
 Executable=/bin/CrashDaemon
 KeepAlive=1
 User=anon
+
+# Uncomment to test what CI does
+#[TestRunner@ttyS0]
+#Executable=/bin/Shell
+#StdIO=/dev/ttyS0
+#Arguments=-c /home/anon/tests/run-tests-and-shutdown.sh
+#Environment=TERM=xterm PATH=/bin:/usr/bin:/usr/local/bin
+#BootModes=text
```

Comment:
This belongs in a documentation file (manpage or `Documentation/`) rather than commented out in the config IMO.

Reply:
That's a good point. I'll look around to see which documentation it belongs in, and if there's no obvious home i'll write up a short doc on how to run the tests in serenity in text mode locally.

Reply:
Added RunningTests.md